cmd/zc_traverser_blob.go

```diff
@@ -161,6 +162,7 @@ func (t *blobTraverser) traverse(preprocessor objectMorpher, processor objectPro
 				preprocessor,
 				getObjectNameOnly(blobInfo.Name),
 				relativePath,
+				common.EEntityType.File(),
 				blobInfo.Properties.LastModified,
```

Comment:
Here's a thought: Blobfuse and ADLSG2 support pseudo folders on blob with a blob that has the metadata property hdi_isfolder = true.

Since blob is semi-capable of being folder aware in that way, should we consider supporting those pseudo folders?

IMO it would make sense to do so, especially if a user has blobs under these pseudo folders.

Reply:
at a bare minimum, we should not enumerate these blobs.

at most, we should copy their metadata as well.

---
In reply to: [376626007](https://github.com/Azure/azure-storage-azcopy/pull/824#discussion_r376626007) [](ancestors = 376626007)

Reply:
I however, don't believe that this would make blob something that empty folders can be copied to, considering how application specific it is.

---
In reply to: [376626284](https://github.com/Azure/azure-storage-azcopy/pull/824#discussion_r376626284) [](ancestors = 376626284,376626007)

Reply:
Tracked already as a separate backlog item.